.semaphore/semaphore.yml

```
@@ -50,6 +50,8 @@ blocks:
             - "sh ./install-nix --no-daemon"
             - ". $HOME/.nix-profile/etc/profile.d/nix.sh"
 
-            - "nix build -f release.nix --print-build-logs --no-link"
+            - "echo 'experimental-features = nix-command flakes' | sudo tee -a /etc/nix/nix.conf"
```

```suggestion
            - "sudo mkdir -p /etc/nix"
            - "echo 'experimental-features = nix-command flakes' | sudo tee -a /etc/nix/nix.conf"
```

Alternatively you can make the installer write this config by setting [`NIX_EXTRA_CONF`](https://github.com/NixOS/nix/blob/ed38c9de00b9b491b6a4c8c9cebb62e1f569dad9/scripts/install-multi-user.sh#L966-L973)